src/pages/FolderPage.js

```diff
@@ -0,0 +1,18 @@
+import FolderList from "../components/FolderPage/FolderList";
+import Header from "../components/Header";
+import Footer from "../components/Footer";
+import AddLinkBar from "../components/FolderPage/AddLinkBar";
+import SearchBar from "../components/SearchBar";
+
+const FolderPage = () => {
+  return (
+    <>
+      <Header />
+      <AddLinkBar />
+      <SearchBar />
+      <FolderList />

```

Comment:
내부 마크업을 보니 이 부분이 `div`로 되어 있네요. 조금이라 더 시멘틱한 마크업을 하기 위해서 해당 부분을 `main` 태그안에 넣어줘야합니다.

```suggestion
<main>
      <AddLinkBar />
      <SearchBar />
      <FolderList />
</main>

```